pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1289,6 +1323,9 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
                 await this.writeFile(`/etc/init.d/buildkitd`, SERVICE_BUILDKITD_INIT, 0o755);
                 await this.writeFile(`/etc/conf.d/buildkitd`, SERVICE_BUILDKITD_CONF);
               }),
+              this.progressTracker.action('Proxy Config Setup', 50, async() => {
```

Comment:
Does this mean we always going to see `Proxy Config Setup` in the progress bar even if there is no proxy settings provided by the user?

Reply:
Yes, but this is not noticeable since the command is executed quite quickly (just writing to config files on WSL side).
Even if there is no proxy configuration I prefer to create an empty `/etc/moproxy/proxy.ini` file to avoid any issue with `moproxy` that can crash when run with a non existing file as an argument.